src/modules/user/kyc/addresses/selectors.ts

```diff
@@ -0,0 +1,8 @@
+import { RootState } from '../../../index';
```

Comment:
Please, add tests for selectors

Reply:
We never cover selectors with tests. It makes no sense at all.

Reply:
It's necessary to be sure that all selectors return correct value from Redux state

Reply:
That's why you need to cover it by tests

Reply:
We didn't do this earlier and it was not correct way

Reply:
Please add tests for selectors